modules/appeals_api/app/swagger/higher_level_reviews/v0/swagger.json

```diff
@@ -947,8 +1060,50 @@
                                 },
                                 "claimant": {
                                   "type": "object",
-                                  "description": "If the claimant is a non-Veteran, claimant information is required. Without this information, we will be unable to identify the claimant. If you're a healthcare provider or an employee of a healthcare provider requesting review of a VA payment decision, you must identify the healthcare provider as the claimant and complete all relevant information in the claimant object.",
+                                  "description": "If the claimant is a not a veteran, claimant information is required. Without this information, we will be unable to identify the claimant. If you're a healthcare provider or an employee of a healthcare provider requesting review of a VA payment decision, you must identify the healthcare provider as the claimant and complete all relevant information in the claimant object.",
```

Comment:
I noticed a typo in each place where this description was updated, "is a not a veteran" when it should be "is not a Veteran."

Reply:
Thank you, good catch! I've fixed these